cosmwasm/contracts/v1/compute-tests/bench-contract/src/msg.rs

```diff
@@ -27,8 +31,44 @@ pub enum ExecuteMsg {
     BenchAllocate {},
     BenchReadLargeItemFromStorage {},
     BenchWriteLargeItemToStorage {},
+    CreateViewingKey {},
+    SetViewingKey {
+        key: String,
+        padding: Option<String>,
+    },
+}
+
+#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, Eq, JsonSchema)]
+#[serde(rename_all = "snake_case")]
+pub enum QueryWithPermit {
```

Comment:
Before reading the implementation of the enum I was sure the QueryWithPermit is a struct that contains a query and a permit. Would suggest to change here the name to "QueryType"